cmd/process-agent/main.go

```diff
@@ -4,11 +4,49 @@ package main
 
 import (
 	"flag"
+	"github.com/DataDog/datadog-agent/cmd/agent/common"
+	"github.com/DataDog/datadog-agent/pkg/api/util"
+	"log"
 	_ "net/http/pprof"
+	"strings"
 
 	"github.com/DataDog/datadog-agent/cmd/process-agent/flags"
 )
 
+func setupConfig() {
+	verb, key, value :=
+		strings.ToLower(flag.Arg(1)),
+		strings.ToLower(flag.Arg(2)),
+		flag.Arg(3)
+
+	// Configure a session token so nothing gets rejected
+	err := util.SetAuthToken()
+	if err != nil {
+		log.Fatal(err)
+		return
+	}
+	settingsClient, err := common.NewSettingsClient()
+
+	if err != nil {
+		log.Fatal(err)
+		return
+	}
+
+	if verb == "get" {
+		get, err := settingsClient.Get(key)
+		if err == nil {
+			log.Println(get)
+		}
+	} else if verb == "set" {
+		_, err = settingsClient.Set(key, value)
+	} else {
+		log.Println("Usage: process-agent config {get setting, set setting value}")
```

Comment:
Because the process-agent binary could be named differently on different OSs, can/should we use `os.Args[0]`?